packages/cpp/src/browser/cpp-client-contribution.ts

```diff
@@ -45,7 +45,7 @@ export class CppClientContribution extends BaseLanguageClientContribution {
             const ERROR_MESSAGE = "Error starting C/C++ language server. " +
                 "Please make sure 'clangd' is installed on your system. " +
                 "You can refer to the clangd page for instructions.";
-            this.messageService.error(ERROR_MESSAGE, READ_INSTRUCTIONS_ACTION).then(selected => {
+            this.messageService.error(ERROR_MESSAGE, undefined, READ_INSTRUCTIONS_ACTION).then(selected => {
```

Comment:
you see, it's inconvenient to add this `undefined` parameter all over the places.

Reply:
agree, no more needed to use undefined with the overload method